distributed/tests/py3_test_asyncio.py

```diff
@@ -0,0 +1,167 @@
+# flake8: noqa
+
+import pytest
+
+asyncio = pytest.importorskip('asyncio')
+aiohttp = pytest.importorskip('aiohttp')  # pytest-aiohttp
+
+import sys
+from operator import add
+from dask import delayed
+from toolz import isdistinct
+from distributed.deploy import LocalCluster
+from distributed.utils_test import gen_cluster, inc, div, slowinc, slowadd, slowdec, randominc
+
+from distributed.asyncio import AioClient, AioFuture
+from distributed import Client, Worker, Scheduler
+
+from tornado.platform.asyncio import to_asyncio_future, BaseAsyncIOLoop
+from tornado.ioloop import IOLoop
+
+
+@delayed
+def decrement(x):
+    return x - 1
+
+
+async def test_asyncio_submit(loop):
+    async with AioClient(loop=loop, processes=False) as c:
+        x = c.submit(inc, 10)
+        assert not x.done()
+
+        assert isinstance(x, AioFuture)
+        assert x.client is c
+
+        result = await x.result()
```

Comment:
Ah I see, you've renamed all of the coroutines to top-level and removed all of the blocking methods. 

Are there cases when someone will want an AioClient for which they sometimes use blocking functions?

Should be concerned about people becoming confused that this client uses the different scheme for naming methods?